README.md

```diff
@@ -98,13 +98,13 @@ Some options are common to all resource types, and can be set with
 `spark.read.format("com.cognite.spark.datasource").option("nameOfOption", "value")`.
 
 The common options are:
-- `apiKey`: *REQUIRED IF NO BEARER TOKEN* A Cognite Data Platform [API key](https://doc.cognitedata.com/dev/guides/iam/authentication.html#api-keys) to be used for authorization.
-- `bearerToken`: *REQUIRED IF NO API-KEY* A Cognite Data Platform [token](https://doc.cognitedata.com/dev/guides/iam/authentication.html#tokens) to be used for authorization.
-- `type`: *REQUIRED* The Cognite Data Platform resource type. See below for more information.
+- `apiKey`: *REQUIRED IF NO BEARER TOKEN* A Cognite Data Fusion [API key](https://doc.cognitedata.com/dev/guides/iam/authentication.html#api-keys) to be used for authorization.
+- `bearerToken`: *REQUIRED IF NO API-KEY* A Cognite Data Fusion [token](https://doc.cognitedata.com/dev/guides/iam/authentication.html#tokens) to be used for authorization.
+- `type`: *REQUIRED* The Cognite Data Fusion resource type. See below for more information.
 - `maxRetries`: The maximum number of retries to be made when a request fails. The default value is 10.
 - `limit`: The number of items to fetch for this resource type to create the DataFrame. Note that this is different
 from the SQL `SELECT * FROM ... LIMIT 1000` limit. This option specifies the limit for the items to be fetched from
-the data platform, *before* filtering and other transformations are applied to limit the number of results.
+the data fusion, *before* filtering and other transformations are applied to limit the number of results.
```

Comment:
Is this how we like to write it? "fetched from the data fusion"? I would prefer "fetched from Cognite Data Fusion", "fetched from CDF" or just "fetched from Data Fusion".